### ISSUE TITLE
Bot is following users already following the main account

### ISSUE THREAD

**Issue by AndreaCorallo** [mifi/SimpleInstaBot#187]
**To Reproduce**
Normal operation?

**Desktop (please complete the following information):**
 - OS: GNU/Linux
 - Browser: appimage vesion
 - Version 1.11.1

**Additional context**
The bot is standard operation mode is following users that are already following the main account.  Is this expected?

**Comment by anon-erpk** [comment]
Yeah, I think it should be possible to not follow who already is a follower
